source/_components/sensor.sensehat.markdown

```diff
@@ -35,4 +35,112 @@ Configuration variables:
   - 'humidity'
   - 'pressure'
 
+___
 
+#### Customizing the Sense HAT data
+
+**Format the sensor values**
+Add the following to your `sensors.yaml`
```

Comment:
Splitting the configuration is an advanced topic. The configuration samples in the documentation should be easy to copy to the `configuration.yaml` file.